src/Umbraco.Core/Models/Email/NotificationEmailAddress.cs

```diff
@@ -0,0 +1,18 @@
+namespace Umbraco.Cms.Core.Models.Email
+{
+    /// <summary>
+    /// Represents an email address used for notifications. Contains both the address and its display name.
+    /// </summary>
+    public class NotificationEmailAddress
+    {
+        public string DisplayName { get; }
+
+        public string Adress { get; }
```

Comment:
Typo here - `Address`.

Reply:
Ahh good spot, for some reason I always misspell Address 😄